gcp-observability/src/main/java/io/grpc/gcp/observability/GcpObservability.java

```diff
@@ -69,7 +69,7 @@ public static synchronized GcpObservability grpcInit() throws IOException {
       GlobalLoggingTags globalLoggingTags = new GlobalLoggingTags();
       ObservabilityConfigImpl observabilityConfig = ObservabilityConfigImpl.getInstance();
       Sink sink = new GcpLogSink(observabilityConfig.getDestinationProjectId(),
-          globalLoggingTags.getLocationTags(), globalLoggingTags.getCustomTags(),
+          globalLoggingTags.getLocationTags(), observabilityConfig.getCustomTags(),
```

Comment:
This only adds custom tags for logging. Don't we need to add same set of custom tags for metrics and traces as well?